src/core.jl

```diff
@@ -127,7 +132,7 @@ function send_value!(node::Signal, x, timestep)
     # Set the value and do actions
     node.value = x
     for action in node.actions
-        do_action(action, timestep)
+        action_queue[action.recipient.value] = action
```

Comment:
`action_queue` keys must be `Signal`s so: `MethodError: Cannot `convert` an object of type Void to an object of type Reactive.Signal{T}`

But I don't think it will happen, unless someone is doing something with non-exported functions/variables. Actions only get created and added to `node.actions` in `add_action!` (which is a nice feature of the existing design btw - made this change easy).